templates/4a_compensate_image_data.ipynb

```diff
@@ -38,9 +38,8 @@
     "import shutil\n",
```

Comment:
<p>"...what signal is being removed from the Noodle channel, which is the main source of noise in most images"</p><p> </p><p>If you are happy with the result of the rosetta compensation, you can move forward with the cells below to process your entire cohort. However, if you would like to adjust the amount of noise being removed, you should instead run the fine tuning notebook, then continue once you have an updated rosetta matrix."</p><p> </p><p>Or something like that</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/212/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>